application/src/main/java/dev/ikm/komet/app/App.java

```diff
@@ -989,20 +955,10 @@ private void generateMsWindowsMenu(BorderPane kometRoot, Stage stage) {
     }
 
     private void showWindowsAboutScreen() {
-        Stage aboutWindow = new Stage();
-        Label kometLabel = new Label("Komet");
-        kometLabel.setFont(new Font("Open Sans", 24));
-        Label copyright = new Label("Copyright \u00a9 " + Year.now().getValue());
-        copyright.setFont(new Font("Open Sans", 10));
-        VBox container = new VBox(kometLabel, copyright);
-        container.setAlignment(Pos.CENTER);
-        Scene aboutScene = new Scene(container, 250, 100);
-        aboutWindow.setScene(aboutScene);
-        aboutWindow.setTitle("About Komet");
-        aboutWindow.show();
+        AboutDialog aboutDialog = new AboutDialog();
+        aboutDialog.showAndWait();
```

Comment:
You can call initOwner on the dialog, passing in an existing window from Komet, that way we avoid an extra window in the taskbar.
Do that before showing the dialog.